.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+known_third_party = cartopy,cdsapi,cftime,coloredlogs,ee,geemap,geopandas,google,googleapiclient,indicators,matplotlib,netCDF4,numpy,pandas,pygsheets,rasterio,rasterstats,requests,rioxarray,scipy,seaborn,shapely,tqdm,utils,utils_evaluation,xarray,xskillscore,yaml
```

Comment:
Is this really needed, or can isort figure this out on its own? 

If we do need it, it would be good to split it up into multiple lines if possible, and also add a blank line at the end.

Reply:
this was only because of the isort-config, so removed now